### ISSUE TITLE
jshttp.github.io and pillarjs.github.io

### ISSUE THREAD

**Issue by timaschew** [pillarjs/pillarjs.github.io#2]
why there are two orgs?

**Comment by q2dg** [comment]
Urge a clear public explanation for this issue: "what is Pillarjs?" "what is Jshttp?"...Something like  Doug said: "jshttp org contains stuff that is too low-level for people to actually use. they are the http-related utility things. pillarjs, on the other hand, is for the more end-user items like routers and the like. soon there will be enough modules in pillarjs to build your own express. that's basically the level pillarjs is, while jshttp is the "boring" things". A simple paragraph in respective main pages could be enough

Another question is what is Expressjs's github's role on this.

**Comment by Fishrock123** [comment]
expressjs is for "official" express-like (node.js) middleware

**Comment by q2dg** [comment]
Yes...but...how long? I mean...is there the "official" purpose of replacing it with pillarjs/jshttp in a near future?

**Comment by Fishrock123** [comment]
Nope. Expressjs will remain as it has been for the foreseeable future.

**Comment by dougwilson** [comment]
Oops, this issue should be closed, since it was a bug with the website (that has since been fixed), so don't mind me closing this issue.

As for @q2dg , I just created a tacking ticket to add a feature to be able to add some kind of blurb/description to the badgeboard site at https://github.com/repo-utils/badgeboard/issues/29

> jshttp

This contains stuff that is low-level, and most end-users won't be using them directly (they are for those who write servers directing using the code `http` module or other framework authors). They are the http-related utility things.

> pillarjs

Is for the more end-user items like routers and the like. Soon there will be enough modules in pillarjs to build your own express (and, express will only contain dependencies on these modules).

> expressjs

This is the organization for express (missing `express` itself, because of various really lame things that have happened; https://github.com/strongloop/express/issues/2310 is just one of the many issues). Currently is contains all the middleware that is "Express-like"; they can be used with `express`, with `connect`, or even with very plain core `http` servers (if they only work with Express, it's a bug).
